doc/man5/flux-config.rst

```diff
@@ -6,21 +6,27 @@ DESCRIPTION
 ===========
 
```

Comment:
"documentaition" is misspelled in the commit title of 07457e450b9f8f4be16445fc15f62f2191095c9a